test/test_helper.rb

```diff
@@ -2,6 +2,7 @@
 
 $LOAD_PATH.unshift File.expand_path("../../lib", __FILE__)
 require "gel"
+require "gel/version"
```

Comment:
I think we should probably do this inside `gel.rb`, so the version is defined for all applications that can see the `Gel` constant.. what do you think?

Reply:
Okay, I agree 👌

Reply:
@matthewd fixed